products/ansible.md

```diff
@@ -29,13 +29,25 @@ releases:
     release: 2020-09-15
     eol: true
     latest: "2.10.7"
+  - releaseCycle: "2.9"
+    release: 2019-10-31
+    eol: false
+    latest: "2.9.27"
 
 ---
 
-> [Ansible](https://ansible.com) is an open-source software provisioning, configuration management, and application-deployment tool enabling infrastructure as code. It runs on many Unix-like systems, and can configure both Unix-like systems as well as Microsoft Windows.
+> [Ansible](https://ansible.com) is an open-source software provisioning, configuration management and application-deployment tool enabling infrastructure as code. `ansible` extends the basic `ansible-core` with additional modules by delivering several collections in an easy-to-consume PyPI package.
 
-The Ansible community package (which this page tracks) typically gets 2 major releases every year. A new minor version is released every 3 weeks. Maintenance fixes are guaranteed for only the latest major release. `ansible-core`, the package meant for developers, has [a separate maintenance policy](https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#id14).
+The `ansible` community package typically gets 2 major releases every year. A new minor version is released every 3 weeks. Maintenance fixes are guaranteed for only the latest major release.
 
 See the [Ansible Roadmap][roadmap] for upcoming release details.
 
-[roadmap]: https://docs.ansible.com/ansible/latest/roadmap/ansible_roadmap_index.html
+[roadmap]: https://docs.ansible.com/ansible/devel/roadmap/ansible_roadmap_index.html
+
+## Python Compatibility
+
+| ansible Version | Minimum Python Version (controller) | Minimum Python Version (modules) |
+|-----------------|-------------------------------------|----------------------------------|
+| 2.9             | 2.7 or 3.5                          | 2.6 or 3.5                       |
+| 2.10            | 2.7 or 3.5                          | 2.6 or 3.5                       |
```

Comment:
```suggestion
```
2.10 is also EOL.

Reply:
Sorry, I keep getting confused with ansible / ansible-base / ansible-core versions 2.9 and 2.10.